app/components/Products.jsx

```diff
@@ -0,0 +1,36 @@
+import React, { Component } from 'react'
+import {connect} from 'react-redux'
+
+const mapStateToProps = (state) => {
+	return { products: state.products.products }
+}
+
+
+export class Products extends Component {
+
+  constructor(props) {
+    super(props)
+  }
+
+	render() {
+
+    const productDivs = this.props.products.map(function(product) {
+      return (
+        <div key={product.id}>
+          <h4>{product.name}</h4>
+          <img src={product.thumbnail} />
+          <p>Price: ${product.price}</p>
+        </div>
+      )
+    });
+
+		return(
+			<div>
+				{productDivs}
+			</div>
+			)
+	}
+}
+
+
+export default connect(mapStateToProps)(Products)
```

Comment:
could you try to change the spacing in this file to two spaces? i think it's inconsistent with the other files.